articles/aks/security-bulletins/overview.md

```diff
@@ -23,6 +23,33 @@ These updates cover security information related to the following AKS components
 - Azure Kubernetes Service Node Image (AKS Node Image)
 - Azure Kubernetes Service Addons (AKS add-ons)
 
+---
+
+## AKS-2025-0010  Nodes can delete themselves by adding an OwnerReference
+
+**Published Date**: August 15, 2025
+
+### Description
+A security issue has been identified in the Kubernetes NodeRestriction admission controller that could allow node users to delete their own node object by patching it with an OwnerReference to a cluster-scoped resource. If the referenced resource or the node object is deleted, Kubernetes garbage collection may remove the node object.
+This issue arises because node users are authorized to perform create and patch operations, but not delete. A compromised node could exploit this to recreate its node object with modified taints or labels, potentially influencing pod scheduling and gaining control over workloads
+
+### References
+
+- [CVE-2025-5187](https://github.com/kubernetes/kubernetes/issues/133471)
+
+### Affected Components
+
+#### [**AKS Cluster**](#tab/aks-cluster)
+
+**Affected Versions**
+
+- [All supported Kubernetes versions](https://docs.azure.cn/en-us/aks/supported-kubernetes-versions?tabs=azure-cli)
```

Comment:
Is this link correct? It's using `docs` rather than `learn` and is in the China `cn` domain. 

If you intended to use the English version, change the link to: 

`[All supported Kubernetes versions](../supported-kubernetes-versions.md)`